_workinggroups/dataanalysis.md

```diff
@@ -24,9 +24,9 @@ well-defined set of problems.
 [Meetings](https://indico.cern.ch/category/10914)
 [Forum](https://groups.google.com/forum/#!forum/hsf-analysis-wg)
 
-Convenors: Paul Laycock (Belle II), TJ Khoo (ATLAS), Andrea Rizzi (CMS)
+Convenors: Katya Govorkova (CERN), Allison Hall (Fermilab), Paul Laycock (Belle II), TJ Khoo (ATLAS), 
```

Comment:
With this addition some parentheses give the affiliation whereas the others give the experiment of the convener. I thought the guideline here was to provide the experiment to show the diversity/coverage?

Reply:
🤦 Thanks!